_std/defines/preferences.dm

```diff
@@ -0,0 +1,5 @@
+#define NAME_CHAR_MAX 16
```

Comment:
this and savefile could probably be combined

Reply:
done!